src/tools/compiletest/src/runtest.rs

```diff
@@ -2408,28 +2409,22 @@ impl<'test> TestCx<'test> {
             "-modify",
         ];
         let tidy_dir = |dir| {
-            let tidy = |file: &_| {
-                Command::new("tidy")
-                    .args(&tidy_args)
-                    .arg(file)
-                    .spawn()
-                    .unwrap_or_else(|err| {
-                        self.fatal(&format!("failed to run tidy - is it installed? - {}", err))
-                    })
-                    .wait()
-                    .unwrap()
-            };
             for entry in walkdir::WalkDir::new(dir) {
                 let entry = entry.expect("failed to read file");
                 if entry.file_type().is_file()
                     && entry.path().extension().and_then(|p| p.to_str()) == Some("html".into())
                 {
-                    tidy(entry.path());
+                    let status =
+                        Command::new("tidy").args(&tidy_args).arg(entry.path()).status().unwrap();
```

Comment:
You are running the `tidy` command here, but I don't see where the old one is run?

Reply:
I'm not sure what you mean? This [used to be a closure](https://github.com/rust-lang/rust/pull/79370/files#diff-03a0567fa80ca04ed5a55f9ac5c711b4f84659be2d0ac4a984196d581c04f76bL2411) but since it's only called in one place I thought it made sense to just call the command directly.

Reply:
Sorry, got disturbed by the PR name. I thought it was about bringing back tests without the `tidy` command. Nevermind! :)